ui/js/controllers/main.js

```diff
@@ -508,10 +508,8 @@ treeherderApp.controller('MainCtrl', [
         };
 
         const getNewReloadTriggerParams = function () {
-            return _.pick(
-                $location.search(),
-                ThResultSetStore.reloadOnChangeParameters,
-            );
+            const locationSearch = $location.search();
+            return Object.assign({}, ...ThResultSetStore.reloadOnChangeParameters.map((prop) => { if (locationSearch[prop]) { return { [prop]: locationSearch[prop] }; } return false; }));
```

Comment:
Checking the way `getNewReloadTriggerParams()` is used, I think including null values is fine, so this could be simplified to:

```js
return Object.assign(
  {},
  ...ThResultSetStore.reloadOnChangeParameters.map(prop => ({
    [prop]: locationSearch[prop]
  }))
);
```